src/Components/ManageTeam/EmployeeForm.tsx

```diff
@@ -0,0 +1,108 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Button, FormControl, FormHelperText, InputLabel, MenuItem, Select, TextField } from '@mui/material';
+import { useEffect, useState } from 'react';
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import * as Yup from 'yup';
+
+type NewEmployeeData = {
+  login: string;
+  password: string;
+  role: string;
+};
+
+const schema = Yup.object().shape({
+  login: Yup.string().required('Required'),
```

Comment:
Tomek also prepared some validation messages. I'd be a good idea to make your and his validation messages uniform if there was time to do so.